src/components/Select.jsx

```diff
@@ -1,29 +1,30 @@
-"use client";
-import { useState } from "react";
-import styles from "../partials/_select.module.scss";
+'use client';
+import { useState } from 'react';
+import styles from '@/partials/_select.module.scss';
 
 export default function Select() {
   const [open, setOpen] = useState(false);
-  const [selected, setSelected] = useState("to do");
-  const [array, setArray] = useState(["to do", "doing", "done"]);
+  const [selected, setSelected] = useState('to do');
+  const [columns, setColumns] = useState(['to do', 'doing', 'done']);
 
   return (
     <div className={styles.select}>
-      <div onClick={() => setOpen(!open)} className={styles["select-trigger"]}>
+      <div onClick={() => setOpen(!open)} className={styles['select-trigger']}>
         <span>{selected}</span>
         <div className={styles.arrow}></div>
       </div>
-      <div className={`${styles["custom-options"]} ${open && styles.open}`}>
-        {array.map((item) => {
+      <div className={`${styles['custom-options']} ${open && styles.open}`}>
+        {columns.map(column => {
           return (
             <span
+              key={column}
```

Comment:
Whenever you iterate and render something, you need to define a `key` prop to avoid unintended side effects.